istioctl/cmd/istioctl/convert/testdata/v1alpha1/myservice-ingress.yaml

```diff
@@ -4,6 +4,7 @@ metadata:
   annotations:
     kubernetes.io/ingress.class: istio
   name: simple-ingress
+  namespace: default
```

Comment:
Just a question: if the namespace is default, is it required to specify that as well?

Reply:
it is not when you submit to kubernetes. this is going to the conversion tool.